schema/schema.go

```diff
@@ -489,6 +489,17 @@ MergeAction:
 
 //JSON returns json format of schema
 func (schema *Schema) JSON() map[string]interface{} {
+	// TODO check if there exist such a function already
+	// if not maybe put this in separate function
+	actions := map[string]interface{}{}
+	for _, a := range schema.Actions {
+		actions[a.ID] = map[string]interface{}{
```

Comment:
I see why you prefer jsonn style over straight GO struct marshal, but it's much easier to achieve with tags, https://golang.org/pkg/encoding/json/#Marshal http://stackoverflow.com/questions/11693865/lower-case-key-names-with-json-marshal-in-go